docs/reference/technology/connectivity/connectivity.md

```diff
@@ -32,41 +35,41 @@ Our Socket API standardizes all of our connectivity options. It supports both IP
 
 Mbed OS provides network drivers, such as Ethernet, Wi-Fi and cellular.
 
-You can find descriptions of supported interfaces and protocols in the [IP networking](ip-networking.html) section.
+You can find descriptions of supported interfaces and protocols in the [IP networking](#ip-networking.html) section.
```

Comment:
This is a page line, you can't use # in it. It's just missing the the ../book-name/ bit